lib/pub_sub/version.rb

```diff
@@ -1,3 +1,3 @@
 module PubSub
-  VERSION = '0.0.7'
+  VERSION = '1.0.0'
```

Comment:
It's worth considering whether the current changes are not an alpha/beta version or a release candidate?